src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] storage;
+    private int capacity;
+    private int size;
+    private int threshold;
+
+    private static class Node<K, V> {
+        private final int hash;
```

Comment:
this field is redundant in the scope of this task
no need to store hash in `Node` objects